packages/schemas/tables/user_social_identity_relations.sql

```diff
@@ -0,0 +1,74 @@
+/* init_order = 2 */
+
+create table user_social_identity_relations (
```

Comment:
add a comment to indicate this relation should never be manually updated?

note: the current gen script only supports one-liner for table comments.

Reply:
added